epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,432 @@ details:
+
+The second step is not required and a `channel_solo_close` could be followed
+either by zero, one or more `channel_slash` transactions every later one
+presenting a newer state and overwriting the previous one. Those are settled by
+a `channel_settle` transaction that finally closes the channel. Let's discuss
+those in detail.
+
+#### Payload and proof of inclusion
+Idea behind `channel_solo_close` and `channel_settle` is for parties to
+provide on-chain the latest channel internal state so the channel can be closed.
+First comes the `channel_solo_close` that provides some off-chain state. Then a
+`channel_slash` can be posted but it is checked that it has a newer state than the
+`channel_solo_close` one. Then parties can post more `channel_slash` transactions
+but those are always checked to be containing a newer channel state than the last
+received on-chain. If one party tries to cheat by posting some old state - the other
+party can present to the chain a newer channel state and this overwrites the previous
+posted one. Thus the comparison on channel states is important. This is done
+by comparing rounds.
+
+Both `channel_solo_close` and `channel_slash` contain a `payload` field. This
+is either a binary containing an `channel_offchain` transaction or an empty
+binary.
+
+If it is a `channel_offchain` transaction - it must be co-signed by both
+participants. It also contains a `channel_id`, `round` and `state_hash`.
+The `channel_id` with a combination with the correct singatures verifies that
+this off-chain transaction indeed is part of the channel off-chain state. The
+`round` represents the height of the channel's state at the time the
+transaction was co-signed by the parties. The higher the round - the newer the
+transaction is. This `round` must be greater than the last on-chain one for that channel.
+The `state_hash` is the internal channel state tree root hash
+at that `round` height.
+
+If the transaction's `payload` is empty - then the latest on-chain state for
+this channel is used. Both `channel_deposit` and `channel_withdraw`
+transactions contain a `round` and a `state_hash` and the latest received one
+overwrites the previous one. If there had been none of those, then the
+`channel_create` transaction is used: it has a `state_hash` and an implicit
+`round = 1`.
+
+Either by having a value in the `payload` or not having one, the
+`channel_solo_close` and `channel_slash` provide a channnel's `round` and a `state_hash`.
+In order to determine the order of the channel's states received - we compare
+the `rounds` and keep the state with the greatest `round`, considered to be
+the _newest_ and _latest_ state. They also provide the `state_hash` the
+channel's state tree had at this `round`.
+
+Both `channel_solo_close` and `channel_slash` contain a `poi` field. This is
+the proof of inclusion for participants' balances in the channel state: all
+the insignificant data in the channel's MPT is replaced by corresponding hashes.
```

Comment:
"MPT (Matricia Perkel Tree)" ;-P
Is the term explained somewhere, including how PoI is managed?